src/components/StartPrjModal/CreateUsingProjectTemplate/index.tsx

```diff
@@ -0,0 +1,229 @@
+      (resource: { title: string }) => ({ title: resource.title })
+    );
+
+    const flProjectCategory = categoriesFromProjectTemplate.map(
+      (category: { categoryName: string; percentageAllocation: number }) => ({
+        categoryName: category.categoryName,
+        percentageAllocation: category.percentageAllocation,
+      })
+    );
+
+    const {
+      flProjectTemplateId,
+      smartContractId,
+      projectTaskContract,
+      projectStatus,
+      createdAt,
+      updatedAt,
+      flProjectCategoryTemplate,
+      flResourcesTemplate,
+      ...temp
+    } = currentTemplate;
+
+    createProjectFromTemplate.mutate({
+      ...temp,
+      organisationId: orgId,
+      flResources,
+      flProjectCategory,
+    });
+  };
+
+  return (
+    <Modal
+      onClose={onClose}
+      width="1000px"
+      height="700px"
+      title="Use Project Template"
+    >
+      <div className={styles[`project-template-modal`]}>
+        <div className={styles[`project-template-modal--picker`]}>
+          <h2>Choose Template</h2>
+          {useFetchProjectTemplates.data?.map(
+            (template: { name: string; flProjectTemplateId: number }) => (
+              <TemplateOption
+                title={template.name}
+                key={getRandomString(5)}
+                id={template.flProjectTemplateId.toString()}
+                filterSelectedTemplate={() => filterSelectedTemplate(template)}
+                selected={
+                  currentTemplate?.flProjectTemplateId ===
+                  template.flProjectTemplateId
+                }
+              />
+            )
+          )}
+        </div>
+
+        {currentTemplate && (
+          <div className={styles[`project-template-modal--preview`]}>
+            <div className={styles[`gif-preview`]}>
+              <img src={gif} alt="template preview" />
+            </div>
+            <h2 className={styles[`template-name`]}>{currentTemplate?.name}</h2>
+            <div className={styles[`text-content`]}>
+              <p>{currentTemplate?.description}</p>
+              <div className={styles.buttons}>
+                <button onClick={createProject}>Get Template</button>
+                <button onClick={() => setShowProjectCreate(true)}>
+                  Skip for now
+                </button>
+              </div>
+            </div>
+          </div>
+        )}
+      </div>
+    </Modal>
+  );
+};
+
+interface ITemplateOptionProps {
+  id: string;
+  title: string;
+  filterSelectedTemplate: (templateId: number) => any;
+  selected?: boolean;
+}
+
+const TemplateOption: FC<ITemplateOptionProps> = ({
+  id,
+  title,
+  filterSelectedTemplate,
+  selected,
+}) => {
+  const onClick = (e: MouseEvent<HTMLElement>) =>
+    filterSelectedTemplate(Number(e.currentTarget.id));
+
+  return (
+    <div
+      className={styles['template-option']}
+      style={{ border: selected ? '0.56px solid #fff' : '' }}
```

Comment:
clsx a better option to use condition
`selected && 'template-option--selected'`